src/instruments/src/SD/Pages/Apu/Apu.tsx

```diff
@@ -0,0 +1,521 @@
+import { SvgGroup } from '../../Common/SvgGroup';
+
+setIsEcamPage('apu_page');
+
+export const ApuPage = () => {
+    const [apuAvail] = useSimVar('L:A32NX_OVHD_APU_START_PB_IS_AVAILABLE', 'Bool', 1000);
+
+    return (
+        <EcamPage name="main-apu">
+            <PageTitle x={283} y={33} text="APU" />
+
+            {/* APU Avail */}
+            {apuAvail
+            && (
+                <SvgGroup x={305} y={85}>
+                    <text x={0} y={0} className="Green FontTitle Center">AVAIL</text>
+                </SvgGroup>
+            )}
+
+            <ApuGen x={105} y={100} />
+
+            <ApuBleed x={420} y={153} />
+
+            {/* Separation Bar */}
+            <SvgGroup x={83} y={252}>
+                <line className="Line Grey" x1={0} y1={0} x2={0} y2={26} />
+                <line className="Line Grey" x1={-1} y1={0} x2={456} y2={0} />
+                <line className="Line Grey" x1={455} y1={0} x2={455} y2={26} />
+            </SvgGroup>
+
+            <NGauge x={155} y={295} />
+
+            <EgtGauge x={155} y={410} />
+
+            <ApuMemos x={370} y={335} />
+        </EcamPage>
+    );
+};
+
+type ComponentPositionProps = {
+    x: number,
+    y: number,
+}
+
+const ApuGen = ({ x, y } : ComponentPositionProps) => {
+    const [apuContactorClosed] = useSimVar('L:A32NX_ELEC_CONTACTOR_3XS_IS_CLOSED', 'Bool', 1000);
+    const [busTieContactor1Closed] = useSimVar('L:A32NX_ELEC_CONTACTOR_11XU1_IS_CLOSED', 'Bool');
+    const [busTieContactor2Closed] = useSimVar('L:A32NX_ELEC_CONTACTOR_11XU2_IS_CLOSED', 'Bool');
+
+    const [apuGenLoad] = useSimVar('L:A32NX_ELEC_APU_GEN_1_LOAD', 'Percent', 500);
+    const [apuGenLoadNormalRange] = useSimVar('L:A32NX_ELEC_APU_GEN_1_LOAD_NORMAL', 'Bool', 750);
+
+    const [apuGenVoltage] = useSimVar('L:A32NX_ELEC_APU_GEN_1_POTENTIAL', 'Volts', 500);
+    const [apuGenPotentialNormalRange] = useSimVar('L:A32NX_ELEC_APU_GEN_1_POTENTIAL_NORMAL', 'Bool', 750);
+
+    const [apuGenFreq] = useSimVar('L:A32NX_ELEC_APU_GEN_1_FREQUENCY', 'Hertz', 500);
+    const [apuGenFreqNormalRange] = useSimVar('L:A32NX_ELEC_APU_GEN_1_FREQUENCY_NORMAL', 'Bool', 750);
+
+    enum apuGenState {STANDBY, OFF, ON}
+    const [currentApuGenState, setCurrentApuGenState] = useState(apuGenState.STANDBY);
+
+    const [apuMasterPbOn] = useSimVar('L:A32NX_OVHD_APU_MASTER_SW_PB_IS_ON', 'Bool', 500);
+    const [apuGenPbOn] = useSimVar('A:APU GENERATOR SWITCH', 'Boolean', 750);
+    const [apuAvail] = useSimVar('L:A32NX_OVHD_APU_START_PB_IS_AVAILABLE', 'Bool', 1000);
+
+    useEffect(() => {
+        if (!apuMasterPbOn && !apuAvail /* FIXME: and apu master switch is valid and apu gen push buttion is valid */) {
+            setCurrentApuGenState(apuGenState.STANDBY);
+        } else if (!(currentApuGenState === apuGenState.STANDBY) && !apuGenPbOn /* FIXME: and apu master switch valid and apu gen pb valid */) {
+            setCurrentApuGenState(apuGenState.OFF);
```

Comment:
**issue**: The APU OFF state is not shown when expected.

![image](https://user-images.githubusercontent.com/1074786/145784788-29949ad4-10c4-4ab2-8e4a-a207ad6c48c4.png)
![image](https://user-images.githubusercontent.com/1074786/145784824-f2979e4e-1345-4780-8bbf-ee7d04a1d3e4.png)

Reply:
Good catch; addressed.